projects/canopy/src/lib/carousel/carousel-item/carousel-item.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, HostBinding } from '@angular/core';
+@Component({
+  selector: 'lg-carousel-item',
+  templateUrl: './carousel-item.component.html',
+  styleUrls: ['./carousel-item.component.scss'],
+})
+export class LgCarouselItemComponent {
+  @HostBinding('class') classes = 'lg-carousel-item lg-padding-md';
```

Comment:
I think at the moment the view encapsulation is stopping the padding styles from working. Generally the approach we take is below, this should set a good baseline. 
```suggestion
  @HostBinding('class.lg-carousel-item') class = true;
```
and then in the css...
```
.lg-carousel-item {
  height: 100%;
  padding: var(--component-padding)
}
```

I think with the current approach there could be a bit of a specificity issue where `lg-padding-md` might not be override-able by the padding directive.

Historically in Angular this `@HostBinding('class')` directly to class would cause issues. It would override classes added by additional directives (in our case utility ones like the Variant one). I think that might be fixed now, but for consistency the above might be nicer.